admin_manual/installation/system_requirements.rst

```diff
@@ -12,31 +12,33 @@ Nextcloud needs a minimum of 128MB RAM, and we recommend a minimum of 512MB.
 Recommended setup for running Nextcloud
 ---------------------------------------
 
-For best performance, stability, support, and full functionality we recommend:
+For best performance, stability, and functionality we recommend:
 
 * Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
 * MySQL/MariaDB
 * PHP 7.0 +
```

Comment:
"7.0+" would make it clear that it isn't 7.0 _and_ something else

Reply:
Or more specific `PHP 7.0, 7.1 or 7.2`

Because we also check for `<7.3`

Reply:
PERFECTION ACHIEVED